geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionAdvisor.java

```diff
@@ -789,43 +788,49 @@ public synchronized long endOperation(long version) {
     return membershipVersion;
   }
 
-  public void waitForCurrentOperations() {
-    long timeout =
-        1000L * this.getDistributionManager().getSystem().getConfig().getAckWaitThreshold();
-    waitForCurrentOperations(timeout);
-  }
-
   /**
    * wait for the current operations being sent on views prior to the joining of the given member to
    * be placed on communication channels before returning
    *
    * @since GemFire 5.1
    */
-  public void waitForCurrentOperations(long timeout) {
+  public void waitForCurrentOperations() {
+    long timeout =
+        1000L * this.getDistributionManager().getSystem().getConfig().getAckWaitThreshold();
+    waitForCurrentOperations(logger, timeout, timeout * 2l);
```

Comment:
"2l" looks like "21". Change it to "2L". This is done in two places. Should a static final constant be created for the multiplier or waitForCurrentOperations be changed to always compute severAlertMS given warnMS?

Reply:
I'm going to change DistributedRegion to use the no-arg version of waitForCurrentOperations().  Then the multiplier is only used in this one place and doesn't need to be exposed as a constant.